mappings/net/minecraft/network/codec/PacketEncoder.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/unmapped/C_hiksfskl net/minecraft/network/codec/PacketEncoder
-	METHOD encode (Ljava/lang/Object;Ljava/lang/Object;)V
+	METHOD encode write (Ljava/lang/Object;Ljava/lang/Object;)V
```

Comment:
```suggestion
	METHOD encode (Ljava/lang/Object;Ljava/lang/Object;)V
```
mapped by #557 as `encode` (you can't name it encode due to an enigma bug that was fixed in 2.2.1)